src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1863,32 +1862,43 @@ def run(self):
     del testcases_metadata
     utils.python_gc()
 
-    fuzz_task_output = uworker_io.FuzzTaskOutput(
+    fuzz_task_output = uworker_msg_pb2.FuzzTaskOutput(
         fully_qualified_fuzzer_name=self.fully_qualified_fuzzer_name,
         crash_revision=str(crash_revision),
         job_run_timestamp=time.time(),
         new_crash_count=new_crash_count,
         known_crash_count=known_crash_count,
         testcases_executed=testcases_executed,
-        job_run_crashes=convert_groups_to_crashes(processed_groups),
     )
+    fuzz_task_output.job_run_crashes.extend(  # pylint: disable=no-member
+        convert_groups_to_crashes(processed_groups))
     if new_targets_count is not None:
-      fuzz_task_output.new_targets_count = new_targets_count
-    return uworker_io.UworkerOutput(fuzz_task_output=fuzz_task_output)
+      fuzz_task_output.new_targets_count = int(new_targets_count)
```

Comment:
It's surprising to me that a variable ending in `_count` is not an int already.

Reply:
since we're reading it from an environment variable, it's a string, so I need to make sure it's properly converted to int.